clang/test/CodeGen/arm-vector_type-params-returns.c

```diff
@@ -0,0 +1,113 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 3
```

Comment:
Given my previous comment, I think it's also worth having a test for:
```
 #include <arm_sve.h>
 #include <arm_neon.h>
```
and
```
 #include <arm_neon.h>
 #include <arm_sve.h>
```
Maybe you can modify this test with some extra RUN lines and `#ifdef` logic for the includes.